fastlane/lanes/build.rb

```diff
@@ -0,0 +1,427 @@
+# frozen_string_literal: true
+
+# Lanes related to Building and Testing the code
+#
+platform :ios do
+  # Builds the WordPress app for Testing
+  #
+  # @called_by CI
+  #
+  desc 'Build WordPress for Testing'
+  lane :build_for_testing do |options|
+    run_tests(
+      workspace: WORKSPACE_PATH,
+      scheme: 'WordPress',
+      derived_data_path: DERIVED_DATA_PATH,
+      build_for_testing: true,
+      device: options[:device],
+      deployment_target_version: options[:ios_version]
+    )
+  end
+
+  # Builds the Jetpack app for Testing
+  #
+  # @called_by CI
+  #
+  desc 'Build Jetpack for Testing'
+  lane :build_jetpack_for_testing do |options|
+    run_tests(
+      workspace: WORKSPACE_PATH,
+      scheme: 'Jetpack',
+      derived_data_path: DERIVED_DATA_PATH,
+      build_for_testing: true,
+      deployment_target_version: options[:ios_version]
+    )
+  end
+
+  # Runs tests without building the app.
+  #
+  # Requires a prebuilt xctestrun file and simulator destination where the tests will be run.
+  #
+  # @option [String] name The (partial) name of the `*.xctestrun` file to run
```

Comment:
I think the original documentation used "partial" because it does not have to be the full base name either. For example, `"foo"` can be used to match `WordPressFooTests.xctestplan` here (which your suggestion would miss to document)

Reply:
I see, good point... I can't suggest anything that feels better at explaining the parameter without becoming verbose. So 🤷‍♂️